app/mailers/application_mailer.rb

```diff
@@ -1,9 +1,15 @@
 class ApplicationMailer < ActionMailer::Base
+  include ActionView::Helpers::SanitizeHelper
+
   default from: ENV.fetch('MAILER_SENDER_EMAIL', 'accounts@chatwoot.com')
-  layout 'mailer'
+  before_action { Current.account = params[:account] }
```

Comment:
What if mail is triggered by a background job (Sidekiq Worker)? It may not have the current account then. Or else we might want to set that somewhere to use this.

Reply:
the idea is to use https://api.rubyonrails.org/classes/ActionMailer/Parameterized.html . So we will be passing in the account at the mailer trigger point.